src/scss/constants.scss

```diff
@@ -98,6 +98,7 @@ $z-layers: (
   navbar-schedule-backdrop: $z-layer-live-schedule,
   navbar-schedule: $z-layer-live-schedule,
   navbar: $z-layer-navbar,
+  navbar__dropdown: -1,
```

Comment:
Why is this set to `-1`? I'm not sure that's what we want to be doing here? I believe what we actually want to do is something similar to what we did with `$z-layer-live-schedule`, and define a new 'layer' for these 'navbar drawers' to sit on. My gut feel/guess is we can probably reuse `$z-layer-live-schedule` here since it's basically just another 'drawer' (though double check that + click around to ensure it still looks right/operates well with everything else). If we do reuse it, it's probably good to rename `$z-layer-live-schedule` to something more generic; perhaps `$z-layer-navbar-drawer`

Reply:
```suggestion
  navbar__drawer: $z-layer-navbar-drawer,
```

Reply:
unfortunately, we can't reuse `$z-layer-live-schedule` (= 20).
Please, check the explanation in my comment : https://github.com/sparkletown/sparkle/pull/1350#discussion_r622827805

but yeah, I can rename it.

Reply:
renamed: https://github.com/sparkletown/sparkle/pull/1350/commits/ea46715895e680fd894657bc141cafd1ce17202b